solutions/usaco-946.mdx

```diff
@@ -264,11 +299,13 @@ by hand when we analyze the edge weight expression and use greedy logic to
 create the optimal grouping. (So the previous solutions will find the exact same
 grouping.)
 
-Senpat's $\mathcal{O}(1)$ code is below:
+Time Complexity: $\mathcal{O}(1)$
 
 
 <LanguageSection>
 
+Senpat's implementation:
```

Comment:
who's senpat

Reply:
@Aryansh-S mentioned him, but idk who he is though